components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css

```diff
@@ -8,8 +8,7 @@
   height: 3rem;
 }
 
-.registerFormPassword-buttonEnabled,
-.registerFormPassword-buttonSessionDisabled {
+.registerFormPassword-buttonEnabled {
```

Comment:
Change "registerFormPassword-buttonEnabled" to `registerFormPassword-enabledButton`

Reply:
fixed